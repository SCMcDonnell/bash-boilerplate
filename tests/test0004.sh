#!/bin/bash

source ../boilerplate.sh

# Write to stdout and log file.
DEBUG_LOG_FILE="test0004_current.log"
DEBUG_SHELL="Yes"
decho "Screen and file."

# Write to stdout only.
DEBUG_LOG_FILE=""
DEBUG_SHELL="1"
decho "Screen only."

# Write to log file only.
DEBUG_LOG_FILE="test0004_current.log"
DEBUG_SHELL="Off"
decho "File only."

# Write nothing
DEBUG_LOG_FILE=""
DEBUG_SHELL="0"
decho "You should never see this line."

