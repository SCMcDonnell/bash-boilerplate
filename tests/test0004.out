Screen and file.
Screen only.
